Create mysterious-halfling.md

malpinder opened the pull request:
Paging @fatbusinessman

malpinder committed:
Create mysterious-halfling.md

AgentAntelope commented on pals/mysterious-halfling.md:
```diff
@@ -0,0 +1,65 @@
+* 10x Shuriken
+* 3x Torches
+* Adventurer's Kit
+* Thieves' tools
```
One adventurer, multiple thieves?

malpinder replied:
Yes, it's like matryoshka dolls, you release a smaller thief into the lock and they have their own heist adventure to break it.

malpinder commented on pals/mysterious-halfling.md:
```diff
@@ -0,0 +1,65 @@
+
+# ??? It Is A Mystery
```
You should come up with a name; I've been using heptambic nearly-alliterative for halfing NPCs (eg Grumblefront Goldenpalm, Riddley Rumblefruit) if you'd like to fit in, but it's fine to be different (halflings come from lots of different countries & cultures, after all)

malpinder commented on pals/mysterious-halfling.md:
```diff
@@ -0,0 +1,65 @@
+Halfing Rogue, Level 1
+
+#### Languages known:
+Common, +1 other
```
You can pick from:
* Dwarven (what dwarves speak)
* Elven (what elves & fey creatures speak)
* Draconic (dragons, kobolds, reptile people etc)
* Giant (giants, orcs, hobgoblins)
* Goblin (goblins, bugbears, and almost every orc & hobgoblin will also know this)
* Deep Speech (mostly-evil things from the Underdark)
* Primordial (elementals and things from Beyond Time)

Whatever you pick will probably hint at a backstory, if/when we get that far.

Commit:
Name the mysterious halfling

Yes, the entire name (and probably a good chunk of the backstory, when that
turns up) is made up so that I can have a piece of corporate business-speak
as my character’s nickname. What of it?

Commit:
Give Moonshot a second language

“Moonshot” sounds sufficiently like something that might be translated badly
from Elvish (assuming an approximately Tolkienesque elvenkind) that there
should probably be some kind of elf-related backstory for him. So, he can
speak Elvish, for reasons yet to be determined.

Commit:
Give Moonshot a silly quote
